Wiki/models/models-aus2200.md

```diff
@@ -0,0 +1,171 @@
+![driving model setup](../Images/AUS2200/driving-model-setup.png)
+
+### `ec_recon_nnn`
+This is the primary driving model reconfiguration task. It creates output files on two domains, the initial AUS2200 domain, and a slightly larger domain of the same resolution that acts as the spatial boundary conditions as the model progresses. One of these tasks is run for each point in the model at which the boundary conditions are synchronised, including the initial and final timesteps. In the default configuration, this means seven tasks per 6 hour cycle. These tasks are not run if the `get_lbc_from_archive` task in the same cycle completed successfully. The exception to this is in the inital cycle, where `ec_recon_000` is run in order to create the initial dump file for the model run.
+
+By default these tasks are configured to run on 2 Sapphire Rapids nodes (`normalsr` queue, 208 cpus) each and complete in approximately 38 minutes. The tasks are independent, and are configured to run in parallel. This block of tasks is the second most expensive task set of the workflow, at a cost of about 1.8kSU/cycle. The output is saved to `{{CYLC_SUITE_SHARE_DIR}}/cycle/{{CYLC_TASK_CYCLE_POINT}}/ec/um`, and can be cleared after the forecast task has been completed.
+
+### `aus2200_d0198_RA3_um_createbc_000`
+This task takes the driving model reconfiguration output and creates the boundary 'frames' that are read by the model every hour to synchronise along the spatial boundaries. These frames are fairly small and data-derived, therefore are ideal for re-use in the 'restart' run mode where spin-up times and model runtimes overlap. This task is not run if the `get_lbc_from_archive` task in the same cycle was successful. The task takes approximately 5 minutes to complete on a single core.
+
+### `put_lbc_to_archive`
+This task archives the boundary conditions created by `aus2200_d0198_RA3_um_createbc_000` to the directory determined above in the `get_lbc_from_archive` task. Note that there is a race condition between testing whether archived boundary conditions are available, to this task running. When running overlapping restart suites, it is best to ensure that the same cycle point is not running in two different suites to avoid this race condition. This task is not run if `Archive LBCS` is set to false in the 'Driving model setup' section of the rose suite configuration (`{{ARCHIVE_LBCS}}` in `rose-suite.conf`)
+
+### `recover_sst`
+This task retrieves the sea-surface temperature from input dump file created by the `ec_recon_000` task. This is necessary as there is an error in the sea ice concentration in the dump file caused by the interpolation from the low-resolution ERA5 dataset, combined with the `era5grib` library's handling of missing values. This can be summarised visually in the following images (Right image created by Martin Dix)
```

Comment:
```suggestion
This task retrieves the sea-surface temperature from input dump file created by the `ec_recon_000` task. This is necessary as there is an error in the sea ice concentration in the dump file caused by the interpolation with the `era5grib` library. This can be summarised visually in the following images (Right image created by Martin Dix)
```

Reply:
My feeling is that it's too much detail better to create and link an issue in era5grib which hopefully eventually will be solved (if it's not due to xesmf) as opposed to put too much detail in here.